.env_example

```diff
@@ -1,7 +1,8 @@
 PYTHONUNBUFFERED=1
-C_FORCE_ROOT=1 # so the celery will not raise an error because of running under root
```

Comment:
Why did you remove the comment?

Reply:
because C_FORCE_ROOT had a value `1 # so the celery will not raise an error because of running under root` not `1`

Reply:
This is only .env_example, so it would have no value at all. 📦 
I think better would be to move the comment one line above, instead of removing it.
But whatever, EOT.